app/routers/system.py

```diff
@@ -118,10 +118,11 @@ def update_software_api(propogate: bool = False):
 
 
 @router.post("/update-and-restart", tags=[p2p_tag])
-def update_and_restart_api(api_key):
+def update_and_restart_api(api_key,env):
```

Comment:
can we put a default environment as `mainnet`?

Reply:
done in latest commit